simul/runfiles/test_randhound.toml

```diff
@@ -0,0 +1,9 @@
+Servers = 10
+Simulation = "RandHound"
+BF = 2
+Rounds = 1
+Trustees = 5
+Purpose = "RandHound Test"
+
+Hosts
+10
```

Comment:
Does it make sense to have more than one round? And perhaps have three lines for the `Hosts`-part: 3, 10, 15 or so.